roles/kubernetes/preinstall/templates/aws-cloud-config.j2

```diff
@@ -0,0 +1,14 @@
+[Global]
+{% if kube_version | version_compare('v1.6', '>=') %}
+zone={{ aws_zone|d("") }}
```

Comment:
i would prefer default(..) since it's used everywhere in the repo. also default vs d is not that big improvement ;) and reads better. wdyt?